index.bs

```diff
@@ -5692,14 +5692,27 @@ useful when there is instance data to track.
 The class would be used in code like:
 
 <pre><code class="lang-javascript">
+  function textDecoderStream() {
+    const td = new TextDecoder();
+    return new TransformStream({
+      transform(chunk, controller) {
+        controller.enqueue(td.decode(chunk));
+      }
+    });
+  }
+
   const data = { userName, displayName, icon, date };
   const ts = new TransformStream(new LipFuzzTransformer(data));
+  const { url } = fetchEvent.request;
 
   fetchEvent.respondWith(
     fetch(url).then(response => {
-      const transformedBody = response.body.pipeThrough(ts);
+      // Decode the binary-encoded response to string
+      const decodedBody = response.body.pipeThrough(textDecoderStream());
```

Comment:
It's better to not create a `decodedBody` variable, but just chain the pipes like this

```javascript
const transformedBody = response.body
  .pipeThrough(new TextDecoderStream())
  .pipeThrough(ts);
```
I don't think it's necessary to comment each operation separately, but you can if you think it is clearer.